libs/api-client/package.yaml

```diff
@@ -0,0 +1,102 @@
+name: api-client
+version: '0.4.2'
+synopsis: (Internal) Wire HTTP API Client
+description: An (internal) client library for the Wire HTTP API.
+category: Network
+author: Wire Swiss GmbH
+maintainer: Wire Swiss GmbH <backend@wire.com>
+license: AGPL-3
+default-extensions:
```

Comment:
Let's use https://github.com/sol/hpack#defaults for this section

Reply:
I didn't make any manual edits since that would take a fair amount more time than just using the automated tool; do you think it's worthwhile to dig through and clean everything up? I suppose if we don't do it now it's unlikely to happen. Should I look into somehow adding some sort of yaml inheritance for these sort of settings or do we not care enough to look into that?

Reply:
You don't need YAML inheritance, it's a built-in hpack feature. You can create a file in the repo:

```
-- package-defaults.yaml

default-extensions:
  - ...
```

And then in every package.yaml file you say on the top level:

```
defaults:
  path: package-defaults.yaml
```

Reply:
Oh, or maybe it should be `local: package-defaults.yaml`; I'm not sure.

Reply:
@ChrisPenner if you don't to I'm happy to do this.

Reply:
After taking a peak; yup this is a good idea 😄 

I added it.